src/quo2/components/avatars/wallet_user_avatar/style.cljs

```diff
@@ -0,0 +1,30 @@
+(ns quo2.components.avatars.wallet-user-avatar.style
+  (:require [quo2.foundations.colors :as colors]))
+
+(defn container
+  [circle-size circle-color]
+  {:width            circle-size
+   :height           circle-size
+   :border-radius    circle-size
+   :text-align       :center
+   :justify-content  :center
+   :align-items      :center
+   :background-color circle-color})
+
+(defn text
+  [text-color]
+  {:color text-color})
+
+(defn circle-color
+  [customization-color theme]
+  (colors/theme-colors
```

Comment:
actually this should be: `    (colors/custom-color customization-color 50 20)`

whenever there is opacity the custom-color is always the -50 suffix used, it's some logic from the designers color system.

I have a pr open to put this into the colors mechanism anyway - https://github.com/status-im/status-mobile/pull/17567/files#diff-2ad5069d9e72acebf9104695abaf82e7264cc4bfa13187904738f39003b61e9b